### PR TITLE
Add test to load_data_by_post.

### PR DESCRIPTION
**Pull request by jerjou**

### CODE REVIEW

File: bigquery/api/load_data_by_post_test.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+@attr('slow')
+class TestLoadDataByPost(tests.CloudBaseTest):
+    dataset_id = 'ephemeral_dataset'
+    table_id = 'load_data_by_post'
+
+    def test_load_csv_data(self):
+        timestamp = int(time.time())
+        with tempfile.NamedTemporaryFile(suffix=".json") as schema_file, \
```
Comment (jonparrott) [inline bigquery/api/load_data_by_post_test.py]: we have fixtures in `/tests/data`, please use those.

**Comment by jonparrott** [comment]
So, how is this sample different from the load_data_from_csv, load_data_from_json samples?

**Comment by jerjou** [comment]
What load_data_from_json sample? :)
It's different from the load_data_from_csv because it loads it from a local file, rather than from a file in cloud storage.

**Comment by jonparrott** [comment]
gotcha. okay, I think perhaps we should give these better names in the future. Anyway, this LGTM.

**Comment by jerjou** [comment]
Updated to use the fixtures. Merging..

**Pull request merged**